clients/src/main/java/org/apache/kafka/common/metrics/stats/Max.java

```diff
@@ -21,8 +21,15 @@
  */
 public final class Max extends SampledStat {
 
+    private final boolean whetherCanBeNegative;
+
     public Max() {
-        super(Double.NEGATIVE_INFINITY);
+        this(false);
+    }
+
+    public Max(boolean whetherCanBeNegative) {
```

Comment:
A more general approach would be to allow one to pass `initialValue` or something like that.